lib/genesis/helpers.go

```diff
@@ -182,13 +202,22 @@ func generatePalletKeyValue(k string, v map[string]interface{}, res map[string]s
 	return true, nil
 }
 
-func buildRawMap(m map[string]map[string]interface{}) (map[string]string, error) {
+func buildRawMap(m Runtime) (map[string]string, error) {
 	res := make(map[string]string)
-	for k, v := range m {
+	mRefObjVal := reflect.ValueOf(m)
+
+	for i := 0; i < mRefObjVal.NumField(); i++ {
+		v := mRefObjVal.Field(i)
+		vInterface := v.Interface()
+		if v.IsNil() {
+			continue
+		}
+
+		k := mRefObjVal.Type().Field(i).Name
```

Comment:
maybe you could add json struct tags to each field (i.e.

```
Name string `json:"name"`
```

and use

```suggestion
		jsonTag := mRefObjVal.Type().Field(i).Tag.Get("json")
		k := strings.Split(jsonTag, ",")[0]
```

So that:
- we can change field name internally and keep the json struct tag as is
- we can have a lowercase json key without having to unexport a struct field
- I believe the end result is a key value map for json right?

Some thoughts @timwu20 ?

Reply:
Hey @qdm12 ,

Can you help me resolve the issue? I couldn't figure out why not getting an expected value on the test.

In the `lib/genesis/keys.go` file, I changed the value of the constant `babePrefixHex` from a hex of "Babe" to a hex of "babe" as you suggested using the JSON tag.

test `TestInstance_BabeConfiguration_NodeRuntime_WithAuthorities`
  - pass if `babePrefixHex = hex of Babe`
  - fail if `babePrefixHex = hex of babe`  (should not fail.)

Reply:
Hey @qdm12 @timwu20 

In some `genesis-spec.json` files, **`Babe`** is used as the JSON key. 
and in some  `genesis-spec.json` files, **`babe`**  is used as the JSON key.

for now, it's not a good idea to use the JSON tags in Go. We should use the struct fields only.

if you want to use JSON tags. we could Modify the code whenever all the genesis-spec.json files would have the same JSON keys.